src/upload.ts

```diff
@@ -12,13 +12,17 @@ import { buildRequestHeaders, buildRequestUrl } from "./request";
 /**
  * The tus chunk size is (4MiB - encryptionOverhead) * dataPieces, set in skyd.
  */
-const TUS_CHUNK_SIZE = (1 << 22) * 10;
+export const TUS_CHUNK_SIZE = (1 << 22) * 10;
 
 /**
- * A number indicating how many parts should be uploaded in parallel, by
- * default.
+ * Indicates what the default chunk size multiplier is.
  */
-const TUS_PARALLEL_UPLOADS = 2;
+const DEFAULT_TUS_CHUNK_SIZE_MULTIPLIER = 1;
```

Comment:
Can we default to something larger like 3 here? I think that's pretty reasonable.